src/balance/Balance.spec.ts

```diff
@@ -65,4 +66,21 @@ describe('Balance', () => {
     expect(status).toBeInstanceOf(SubmittableResult)
     expect(status.isFinalized).toBeTruthy()
   })
+  it('should make transfer of amount with arbitrary exponent', async () => {
+    const alice = await Identity.buildFromURI('//Alice')
```

Comment:
you should check out the conventional way we are doing the tests.

Having a let at the top. Would be better if we keep things in line with all the other tests.